b'Caused by: javax.resource.ResourceException: IJ000453: Unable to get managed connection for java:jboss/datasources/oracle Caused by: javax.resource.ResourceException: IJ000655: No managed connections available within configured blocking timeout (30000 [ms])\r\n| ERROR |  Connection reset by peer GenericHttpAdapter.java             | 558 ] :-> TXN_ID | 164669870451899_APP1BL03 Exception occured while sending to ResponseServlet :::\r\nxception occured......java.sql.SQLRecoverableException: IO Error: Connection timed out (Read failed) java.sql.SQLRecoverableException: IO Error: Connection timed out (Read failed) at oracle.jdbc.driver.T4CPreparedStatement.executeForDescribe(T4CPreparedStatement.java:792) ~[?:?]\r\nSql exception occured......java.sql.SQLRecoverableException: Closed Connection /164660784511899_APP2BL05 at io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:104) [undertow-servle t-2.0.19.Final.jar!/:2.0.19.Final] at io.undertow.server.Connectors.executeRootHandler(Connectors.java:364) [undertow-core-2.0.19.Final.jar!/:2.0.19.Fina\r\n\r\n\r\n\r\n'